app/controllers/connect_four_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ConnectFourController < ApplicationController
+  def index
+    session[:board] = Array.new(7){Array.new(0)}
+  end
+
+  def make_move
+    @game = ConnectFour.new(params[:move], session[:board])
+    @message = @game.message
+    #if @game.gameover?
+    @board_render = @game.board
+    session[:board] = @game.board
+    render :index
+  end
```

Comment:
Are lines 10 and 11 redundant? Why not just save `@game.board` to the session and reference it from there?